### ISSUE TITLE
openucx.org/documentation/ should use latest available docs instead of using broken links

### ISSUE THREAD

**Issue by quildtide** [openucx/ucx#6493]
As of March 14th 2021:

# Steps to reproduce problem
- Visit https://www.openucx.org/documentation/
- Scroll down to the Documentation section, (or use https://openucx.readthedocs.io/en/master/#documentation)
- Click on the HTML and/or the PDF links for the API Doc
- Enjoy a beautiful 404 error

# Cause
Users are routed towards documentation for v1.10, but documentation for v1.10 does not yet exist.

However, it seems reasonable to me that linking users to documentation for v1.9, which does indeed exist, is preferable over not linking users to documentation at all. An optimal solution under this logic is to always link to the latest API documentation available, even if it's different from the latest API available.

# Solution
Someone has already done the vast majority of the work necessary by setting up the necessary symbolic links in this repository, but a single line still need be changed in order to fix this:

Replace [line 75](https://github.com/openucx/ucx/blame/master/docs/source/index.rst#L75) of the [the relevant file](https://github.com/openucx/ucx/blob/master/docs/source/index.rst), which is currently:
```
*  API doc: `HTML <https://openucx.github.io/ucx/api/v{VERSION}/html/index.html>`_ `PDF <https://openucx.github.io/ucx/api/v{VERSION}/ucx-v{VERSION}.pdf>`_
```

with

```
*  API doc: `HTML <https://openucx.github.io/ucx/api/latest/html/index.html>`_ `PDF <https://openucx.github.io/ucx/api/latest/ucx.pdf>`_
```

I made this rather trivial fix in [a commit in my fork](https://github.com/Thalassocracy/ucx/commit/228d1bd477a1506ff6d1023608b92ce81f0523cc) but I stopped short of a pull request as I am yet to submit a CLA. I am not opposed to submitting a CLA, but I feel that someone who has already submitted a CLA could probably fix this before I would get the counter-signed CLA returned to me.

**Comment by yosefe** [comment]
@Thalassocracy thanks for the report. The issue happened after we released v1.10 version last week and didn't upload the docs for it, and is fixed now.

**Comment by shamisp** [comment]
Seems like this issue was resolved ?
